src/stream_base-inl.h

```diff
@@ -173,7 +173,7 @@ StreamWriteResult StreamBase::Write(
     total_bytes += bufs[i].len;
   bytes_written_ += total_bytes;
 
-  if (send_handle == nullptr) {
+  if (send_handle == nullptr && !skip_try_write) {
```

Comment:
@RafaelGSS This change is an optimization for a corner case, and I think the existing tests are enough to assert its correctness

Reply:
It seems the `skip_try_write` parameter is new. Where is this used?